docs/conf.py

```diff
@@ -177,7 +177,7 @@
 
 # -- Options for LaTeX DoIterationOutput ------------------------------------------
 
-latex_elements = {
+latex_elements: Dict[type, type] = {
```

Comment:
I don't know what this is of type "type", isn't "Dict" good enough?

Reply:
Yes it is.